build.gradle

```diff
@@ -37,31 +37,32 @@ ospackage {
     postUninstall file(resourcesDir + 'deb/control-runtime/postrm')
     
     configurationFile('/etc/default/openhab2')
+    configurationFile('/usr/lib/systemd/system/openhab2.service')
```

Comment:
They are unlikely to change it, but at the moment it is necessary for some users using GPIO bindings. OpenHAB2 works in this case if the service is **started** as root. And this change was requested of me by @ThomDietrich [Here](https://community.openhab.org/t/reset-of-openhab2-service-after-update/18547/4?u=benjy)